index.html

```diff
@@ -48,6 +48,12 @@ <h1 class="pl-heading pl-element-title">{{ post.title }}</h1>
                         <h2 class="pl-heading pl-tab-heading pl-element-preview-title">{{post.title}} Preview</h2>
 
                         <div class="pl-tab-content">
+                            {% if post.tags contains "icons" %}
```

Comment:
This logic needs to be added to the ``templates/elements.html`` file as well. Please sync it up.